pkg/core/serving/knative/servingrun.go

```diff
@@ -347,6 +347,11 @@ func (r *servingRun) createService(s *openfunction.Serving, cm map[string]string
 		template.Containers = append(template.Containers, *container)
 	}
 
+	if _, ok := s.Annotations[constants.WasmVariantAnnotation]; ok && template.RuntimeClassName == nil {
+		runtimeClassName := constants.WasmEdgeRuntimeClassName
```

Comment:
what is the difference between [here](https://github.com/OpenFunction/OpenFunction/pull/415/files#diff-97f08fc10cfa4844d958824ed5b4bfed497db5e1380e51bf401c1674d6ad5089R150-R151) and this?

Reply:
The template maybe generate during servingrun, webhook only handle `r.Spec.Serving.Template != nil` case.